src/jira/client/index.ts

```diff
@@ -302,7 +302,7 @@ const batchedBulkUpdate = async (
 			}
 		};
 		return instance.post("/rest/devinfo/0.10/bulk", body).catch((err) => {
-			logger.error({err, body, data}, "Jira Client Error: Cannot update Pull Request")
+			logger.error({ err, res: err.response, req: err.request, body, data }, "Jira Client Error: Cannot update Pull Request");
```

Comment:
Let's change the message, because it is not only related to pull requests

Reply:
yeah, good point.